### DISCUSSION TITLE
Hierarchy of Classes

### DISCUSSION THREAD

**Discussion by HannahStrimpel** [frc5567/2023-Robot#191] [Team Posts]
![Class Hierarchy](https://user-images.githubusercontent.com/45899961/74692453-3d717e80-51b5-11ea-9d77-546d52025091.jpg)
![IMG-1150](https://user-images.githubusercontent.com/45899961/74692461-44988c80-51b5-11ea-874a-c1532716abe4.jpg)

**Comment by MikasaSucasa** [comment]
@JooshO can you look into where limelightReader is **instantiated**
